Vagrantfile

```diff
@@ -0,0 +1,10 @@
+# -*- mode: ruby -*-
```

Comment:
In order to be useful it'll need to eventually end up in the root of the
project. Vagrant essentially starts a headless vm and shares the directory
the vagrant file is in with it.

It would probably be fine to just include it in util/ and add instructions
to the wiki that include copying it to the root before using it. (I'm
currently without computer now else I'd do just that).
On Jun 18, 2015 12:25 PM, "Kevin Lange" <notifications@github.com> wrote:

> In Vagrantfile
> <https://github.com/klange/toaruos/pull/99#discussion_r32749092>:
>
> > @@ -0,0 +1,10 @@
> > +# -*- mode: ruby -*-
>
> I'm not familiar with how Vagrant works, but I don't really want this at
> the top level of the repository - can we put it in util/ (or maybe
> toolchain/ is a better place...)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/klange/toaruos/pull/99/files#r32749092>.
>